fiware-region-sanity-tests/commons/constants.py

```diff
@@ -42,15 +42,20 @@
 # TASK TIMERS AND TIMEOUTS
 DEFAULT_REQUEST_TIMEOUT = 60
 MAX_WAIT_ITERATIONS = 60
+MAX_WAIT_SSH_CONNECT_ITERATIONS = 35
 SLEEP_TIME = 5
 
 # TEST DATA
 BASE_IMAGE_NAME = "CentOS-6.3init"
 TEST_FLAVOR_DEFAULT = "small"
```

Comment:
A regex is created from this string, so "small" will mean: a flavor named "small" with and optional prefix and "." separator. Doing this way is easier to configure (we avoid the regex complexity here)